From d1e957dd83771577171593ed58add25a7090b2fd Mon Sep 17 00:00:00 2001
From: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
Date: Wed, 7 Aug 2024 15:39:46 +0530
Subject: [PATCH] Update composer aidl version to 3

Test done:
- Android boot in GVT-d and BM config
- adb reboot

Tracked-On: OAM-122191
Signed-off-by: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
---
 groups/device-specific/caas/manifest.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/groups/device-specific/caas/manifest.xml b/groups/device-specific/caas/manifest.xml
index f1419e6..ff961a0 100644
--- a/groups/device-specific/caas/manifest.xml
+++ b/groups/device-specific/caas/manifest.xml
@@ -11,7 +11,7 @@
     </hal-->
     <hal format="aidl">
         <name>android.hardware.graphics.composer3</name>
-        <version>2</version>
+        <version>3</version>
         <interface>
             <name>IComposer</name>
             <instance>default</instance>
-- 
2.45.2

